viral/imaging_utils.py

```
@@ -2,6 +2,7 @@
 
 from scipy.ndimage import gaussian_filter1d
 from typing import List, Tuple
+import os
```

Ruff (0.11.9)

`os` imported but unused

Remove unused import: `os`

(F401)